src/main/java/lotto/LottoApplication.java

```diff
@@ -16,7 +16,12 @@ public static void main(String[] args) {
     String numbersString = InputView.readWinningNumbers();
     Lotto winningNumbers = new Lotto(numbersString);
 
-    LottoMatchResult matchResult = lottos.matchResult(winningNumbers);
+    String bonusNumberString = InputView.readBonusNumber();
+    int bonus = Integer.parseInt(bonusNumberString);
+    LottoNumber bonusNumber = new LottoNumber(bonus);
```

Comment:
```suggestion
    LottoNumber bonusNumber = new LottoNumber(InputView.readBonusNumber());
```
LottoNumber에 String을 인자로 받는 부 생성자를 추가하면 어떨까?